emcc.py

```diff
@@ -1091,6 +1090,8 @@ def add_link_flag(i, f):
     if final_suffix == '.mjs':
       shared.Settings.EXPORT_ES6 = 1
       shared.Settings.MODULARIZE = 1
+
+    if final_suffix in ('.mjs', '.js', ''):
```

Comment:
this list might be a new `JS_ENDINGS` perhaps?

Reply:
Since it only used in the this one place I'd rather keep it obvious and local.